ext/hash/hash_sha_ni.c

```diff
@@ -31,7 +31,7 @@
 
 # include <immintrin.h>
 
-# if PHP_HASH_INTRIN_SHA_RESOLVER
+# if  defined(PHP_HASH_INTRIN_SHA_RESOLVER) && defined(HAVE_FUNC_ATTRIBUTE_TARGET)
```

Comment:
Duplicate space at the beginning:
```suggestion
# if defined(PHP_HASH_INTRIN_SHA_RESOLVER) && defined(HAVE_FUNC_ATTRIBUTE_TARGET)
```

Reply:
Ah, thank you! I'll fix when applying the PR.